app/src/main/res/values/static_strings.xml

```diff
@@ -15,4 +15,5 @@
     <string name="about_debug_menu_toast_progress" translatable="false">Debug menu: %1$d click(s) left to enable</string>
     <string name="about_debug_menu_toast_done" translatable="false">Debug menu enabled</string>
 
+    <string name="number_of_opened_tabs" translatable="false">Number of opened tabs</string>
```

Comment:
Shouldn't this be in [`preference_keys.xml`](https://github.com/mozilla-mobile/focus-android/blob/main/app/src/main/res/values/preference_keys.xml)?